packages/test-app/app/templates/docs/poll-task.md

```diff
@@ -75,29 +77,30 @@ This is where `pollTask` really shines. You could rewrite the above example to u
 like this:
 
 ```js
-import Component from '@ember/component';
-import { inject } from '@ember/service';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { service } from '@ember/service';
```

Comment:
Note: this assumes Ember v4. Seems fine?

Reply:
I think so. Doesn't mean folks can't use it on older versions.